financialaid/models.py

```diff
@@ -149,6 +149,17 @@ class FinancialAid(TimestampedModel):
     date_exchange_rate = models.DateTimeField(null=True)
     date_documents_sent = models.DateField(null=True)
 
+    def to_dict(self):
+        """
+        Get the model_to_dict of self
+        """
+        ret = model_to_dict(self)
+        if self.date_exchange_rate is not None:
+            ret["date_exchange_rate"] = str(ret["date_exchange_rate"])
+        if self.date_documents_sent is not None:
+            ret["date_documents_sent"] = str(ret["date_documents_sent"])
```

Comment:
Please use `isoformat()` here and above. `str` is rarely the right way to convert an object to a string